src/lib/components/side-nav.svelte

```diff
@@ -96,6 +102,21 @@
         <div class="nav-title">{translate('feedback')}</div>
       </NavRow>
     </slot>
+    <NavRow {isCloud} noHover>
+      <NavTooltip right text={labsText}>
+        <button
+          aria-label="labs mode"
```

Comment:
We should add a translate here
```suggestion
          aria-label={translate('labs')
```

Maybe `labs` in common.ts can be `Labs Mode` instead?

Reply:
Or actually, what if we just did this instead?

```suggestion
          aria-label={labsText}
```